Backend/prisma/migrations/20240724072343_init_schedule_table/migration.sql

```diff
@@ -33,8 +33,9 @@ CREATE TABLE "serviceProvider" (
 CREATE TABLE "appointment" (
     "id" TEXT NOT NULL,
     "date" TIMESTAMP(3) NOT NULL,
-    "time" TIMESTAMP(3) NOT NULL,
+    "time" TEXT NOT NULL,
```

Comment:
Why text instead of timestamp?

Reply:
I just hard coded that in to test the endpoint flow. Now that I'm working on bookings and appointments, I'll adjust this as well